client/i18n/locales/english/intro.json

```
@@ -2052,7 +2052,10 @@
       "yshh": { "title": "237", "intro": [] },
       "wyxz": { "title": "238", "intro": [] },
       "dtfv": { "title": "239", "intro": [] },
-      "miza": { "title": "240", "intro": [] },
+      "lab-quick-sort": {
+        "title": "Build a Quick Sort Algorithm",
```

```suggestion
      "lab-quick-sort-algorithm": {
        "title": "Build the Quick Sort Algorithm",
```
Not sure about this, but to me it makes more sense using "the".
Folders should be renamed `lab-quick-sort-algorithm`.